src/sys/time.rs

```diff
@@ -6,6 +6,13 @@ use std::convert::From;
 use std::time::Duration;
 use std::{cmp, fmt, ops};
 
+const fn zero_init_timespec() -> timespec {
+    // `std::mem::zeroed()` is not yet a const fn (https://github.com/rust-lang/rust/issues/62061)
```

Comment:
Another option is `std::mem::MaybeUninit::zeroed()`.  It's also not yet const (https://github.com/rust-lang/rust/issues/91850), but seems to be moving faster.

Reply:
Updated the comment to reflect that.